main/create-window.js

```diff
@@ -86,9 +86,13 @@ module.exports = function createWindow (options) {
   }
 
   function updateTitle () {
-    var prefix = fromFile ? (path.basename(options.filePath) + ' - ') : ''
+    if (options.title) {
+      win.setTitle(options.title)
```

Comment:
@langpavel Yes, i saw that. Do you want to change it so that the ` - vmd` suffix remains? Or was it intentional that it's removed when passing the `--title` option?